MapboxVisionAR/Render/ARScene/ConcreteNodes/ARLaneNode/ARLaneEntity.swift

```diff
@@ -0,0 +1,22 @@
+import simd
+
+/**
+ Implementation of `AREntity` for AR lane.
+ */
+class ARLaneEntity: AREntity {
```

Comment:
What is the purpose of the class? Seems like just a way to configure an entity, if so should it be better to use builder or something like that?

Reply:
Usage builder is one of possible solution. But. In the current implementation there's a dependency on Renderer's state and that's another piece we should update. IMHO that's also not a part of current PR. Corresponding issue #120 was created.
WDYT?

Reply:
You created this class in this pr and you're saying that this is not a part of the current PR means it shouldn't be in the pr

Reply:
I'll try to explain it one more time: yes, formally that class was created "here". But on the other hand it's only adoption for `AREntity` which lived in code before the current PR.

All changes I've made (including creation of "new" classes) was made with only one purpose - to proceed with the initial task easily (see legacy implementation of `ARRenderer`).

IMO all changes that either break API in callers or introduce changes that do not affect initial task, should be done separately. More specifically, if we're going to add a way to configure `ARLane` via builder, we should also should avoid dependency injection to separate process of configuration `AR-smth` object. That affects API on higher levels (`ARRenderer` and its callers, and that's a additional diff eventually). That's why I think such task should be done separately.

Definitely I can do this work here, but do you want to review such changes as one PR? All in all I suggest to prefer continuous integration with smaller changes.

Does that make sense?

Reply:
Agree to get rid of class for now.